modules/api-explorer-client/config/webpack.config.prod.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
     // https://github.com/facebookincubator/create-react-app/issues/290
     // `web` extension prefixes have been added for better support
     // for React Native Web.
-    extensions: ['.web.js', '.mjs', '.js', '.json', '.web.jsx', '.jsx'],
+    extensions: ['.web.js', '.mjs', '.js', '.json', '.web.jsx', '.jsx', '.ts', '.tsx', '.d.ts'],
```

Comment:
I think `d.ts` can be removed from here as it's not being imported from any file.

Reply:
I think we can not remove this because we must handle  `d.ts`  which are coming from other outside dependencies